node/actors/bft/src/replica/state_machine.rs

```diff
@@ -59,58 +58,70 @@ impl StateMachine {
             high_vote: backup.high_vote,
             high_qc: backup.high_qc,
             block_proposal_cache,
-            timeout_deadline: time::Deadline::Infinite,
+            timeout_deadline: sync::watch::channel(time::Deadline::Infinite).0,
         };
         // We need to start the replica before processing inputs.
         this.start_new_view(ctx).await.wrap("start_new_view()")?;
         Ok(this)
```

Comment:
Nit: Maybe we move this part to the run method. Rename this method to `new`.

Reply:
We can still separate `new()` from `start()` (in addition to `run()`), but it's probably too verbose. 

If it's too confusing, I can rename `run()` with something more specific. I chose not to in order to emphasize that it serves as the main loop.